test/testcafe/spec/ChallengeAuthenticatorPhone_spec.js

```diff
@@ -512,42 +527,48 @@ test
   .requestHooks(invalidCodeMock)('Entering invalid passcode results in an error', async t => {
     const challengePhonePageObject = await setup(t);
     await challengePhonePageObject.verifyFactor('credentials.passcode', 'abcd');
-    await challengePhonePageObject.clickNextButton();
+    await challengePhonePageObject.clickVerifyButton();
     await challengePhonePageObject.waitForErrorBox();
-    await t.expect(challengePhonePageObject.getInvalidOTPFieldError()).contains('Invalid code. Try again.');
+    // in v3, the "invalid code" field error is not shown. Test mock issue, not product issue
+    if (!userVariables.v3) {
+      await t.expect(challengePhonePageObject.getInvalidOTPFieldError()).contains('Invalid code. Try again.');
+    }
```

Comment:
If you could create a jira and comment here so we can come back and clean up the mock. We try to avoid adding this v3 conditional in the spec unless there is a true difference between v2 and v3.

Reply:
This is fixed now with the new mock file